modules/ppcp-card-fields/resources/js/Render.js

```diff
@@ -9,7 +10,7 @@ export function renderFields( cardFields ) {
 		cardFields
 			.NameField( { style: { input: styles } } )
 			.render( nameField.parentNode );
-		nameField.hidden = true;
```

Comment:
We should keep this line, so the if-condition works as expected: Each field should be hidden once, and not trigger a `field.render()` call again:

https://github.com/woocommerce/woocommerce-paypal-payments/blob/5237ebc0ed39997fd2abb27e95e97c2dbe78906f/modules/ppcp-card-fields/resources/js/Render.js#L7